packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -2175,6 +2175,26 @@ void main() {
     FeatureFlags: () => TestFeatureFlags(isMacOSEnabled: true),
   });
 
+  testUsingContext('flutter create . on and existing plugin should show "Your example app code in"', () async {
+    Cache.flutterRoot = '../..';
+
+    final CreateCommand command = CreateCommand();
+    final CommandRunner<void> runner = createTestCommandRunner(command);
+    final String projectDirPath = globals.fs.path.normalize(projectDir.absolute.path);
+    final String relativePluginPath = globals.fs.path.normalize(globals.fs.path.relative(projectDirPath));
+
+    await runner.run(<String>['create', '--no-pub', '--org=com.example', '--template=plugin', '--platforms=android', projectDir.path]);
+    expect(logger.statusText, isNot(contains('Your example app code in $relativePluginPath\n')));
```

Comment:
I don't really understand this test. isn't "Your example app code" the phrase you want to see?

Reply:
This test case is exactly what I expected to see. @jmagman requested a test from me, and I provide it. Please correct me if I missing something or wrong.

Reply:
Thanks, I’ll fix that